code/game/objects/structures/watercloset.dm

```diff
@@ -486,8 +486,22 @@
 		new_sink.setDir(dir)
 		qdel(src)
 		return
+	if(I.tool_behaviour == TOOL_WRENCH && !(flags_1 & NODECONSTRUCT_1))
```

Comment:
Check the `wrench_act()` proc. Use it instead of adding it to `attackby()`, please.